.github/workflows/build-image.yml

```diff
@@ -0,0 +1,16 @@
+name: Build Docker image
+
+on: [pull_request]
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Build container image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
```

Comment:
We should also push the container to a tag. Given Docker Hub has been considering cutting its free hosting for a while, I think now would be a good time to move to GHCR, at least temporarily.

Reply:
Alright, that makes sense.